### ISSUE TITLE
Can't map players

### ISSUE THREAD

**Issue by killercode** [bedrock-viz/bedrock-viz#3]
I followed the mcpe_viz.cfg tip to map the players and it's not working, players are always like *Unknown*

Here is a piece of the log
[   info] Unmapped remote player: 22ab1e1d-4159-3867-8f3b-a60cb472ea11
[   info] Unmapped remote player: 33d314f1-5884-3d3f-a55c-d66847f15f55
[   info] Unmapped remote player: 71d453d4-e55c-3a74-87ef-5a769b061113
[   info] Unmapped remote player: 806cbaa5-d1da-37ce-b046-509420b1e99f
[   info] Unmapped remote player: 82dc807b-f314-3b6d-bc6a-e823b67e25af
[   info] Unmapped remote player: e7b0b45e-9684-3508-b36c-b52c620bcb55
[   info] Unmapped remote player: server_48a3f13b-b55c-4d2b-ab0a-223501620826
[   info] Unmapped remote player: server_7a24e15e-5c2d-43e4-9704-13e21f19c7b6
[   info] Unmapped remote player: server_fbb51582-a31c-4585-945c-733af1725977

I tried to only configure the ones that are appearing on the map, which are the "server" ones... 

I created the mapping file:
mcpe_viz.local.cfg:
player-id: server_48a3f13b-b55c-4d2b-ab0a-223501620826 Nick1
player-id: server_fbb51582-a31c-4585-945c-733af1725977 Nick2
player-id: server_7a24e15e-5c2d-43e4-9704-13e21f19c7b6 Nick3

What is wrong here? is this some bug?

Thanks a lot.

**Comment by tomnolan** [comment]
@killercode did you ever get this to work? Let me know if this is still an issue, I'll take a look and see if there is an issue. This could be related to #18 and your config file not actually loading properly.

**Comment by counterfitninja** [comment]
@tomnolan i have not been able to get it to work. 

Any way I can buy you a coffee for all your work getting this updated?

**Comment by tomnolan** [comment]
Hah! No worries, just glad there are people out there who appreciate the updates.

I'll take a look at the player parsing, I thought I had it working on one of my worlds before, I'll have to double check.

**Comment by counterfitninja** [comment]
Do let me know if you want any beta testing. Always happy to help with different size maps

**Comment by cabbey** [comment]
I don't believe the "local" config file logic is working. I've been meaning to look into that.

As a workaround: you can insert those values into the main `data\bedrock_viz.cfg` file (note the name has changed) which I can confirm works with the recent v0.1.3 release. (I have a copy on the side that I've added players and a few geojson-block elements that I specify with a `--config` command line option.)

**Comment by cabbey** [comment]
yeah, the "local" config file wasn't "working" because there doesn't seem to be any code that even tried to do that. :roll_eyes: Fix proposed on the above PR, would love to get some feedback on if the docs make any sense, and if that style of handling it works for you @counterfitninja

**Comment by counterfitninja** [comment]
Thanks for the update @cabbey i had already been editing the previous bedrock_viz.cfg file to add player names and that was working fine, after i realised that on a server you get multiple entries for the play, one to link the xbox account, one local player, and one server player. It is difficult to know who is who on a server as there is pretty much no way to find out which user it which without asking. (i did find other ways todo this via mcctoolchest and guessing their inventory) 

The docs did make sense to me, but is there another way to click a play in the UI and tag them? ( I guess this would need a writable DB somewhere. Would having all the data in a SQLlite DB be better than the xml files, that way it might be writable? )

**Comment by cabbey** [comment]
If I were to build that, I'd suggest that the FE pull from the DB to display the names rather than having the visualizer embed them into the map statically... it would be best to just put the player id in the geojson and leave it to the FE to translate the ID to a name. You could almost envision that as a addon to the produced web app. Perhaps we should think about some sort of extension system that could be added in and allow users to deploy various addon as they need. I'd personally love to see a "real time" version of that where the player location is loaded dynamically from the server instead of from the static map dump.
